src/collection.js

```diff
@@ -134,6 +134,29 @@ function createUUIDSchema() {
   };
 }
 
+/**
+ * IDSchema for when using kinto.js as a key-value store.
+ * Using this IDSchema requires you to set a property as the id.
+ * This will be the property used to retrieve this record.
+ *
+ * @example
+ * const exampleCollection = db.collection("example", { idSchema: createKeyValueIdSchema() })
+ * await exampleCollection.create({ title: "How to tie a tie", favoriteColor: "blue", id: "user123" }, { useRecordId: true })
+ * await exampleCollection.getAny("user123")
+ *
+ */
+
+export function createKeyValueIdSchema() {
```

Comment:
Nit: can we call it maybe `createKeyValueStoreIdSchema`? This is a term of art with a specific meaning, see https://en.wikipedia.org/wiki/Key-value_database.